src/main/java/racingcar/domain/ParticipatedCars.java

```diff
@@ -6,51 +6,35 @@
 import java.util.List;
 
 import racingcar.util.BoundedRandomNumberGenerator;
+import racingcar.util.RandomNumberGenerator;
+import racingcar.validator.exception.CarNameDuplicationException;
 
 public class ParticipatedCars {
-    private static final int MAX_BOUND = 9;
-    private static final int MIN_BOUND = 0;
     private static final int START_POSITION = 0;
 
     private final List<Car> cars;
 
-    public ParticipatedCars() {
+    public ParticipatedCars(List<String> carNames) {
+        checkCarNameDuplicated(carNames);
         this.cars = new ArrayList<>();
-    }
-
-    public void generateCars(List<String> carNames) {
         for (String carName : carNames) {
             cars.add(new Car(carName, START_POSITION));
         }
     }
 
-    public WinnerNames findWinners() {
-        return getWinnerNamesWithFirstCar(findFastestCar());
-    }
-
-    private WinnerNames getWinnerNamesWithFirstCar(Car firstCar) {
-        return WinnerNames.of(this, firstCar);
-    }
-
-    private Car findFastestCar() {
-        Collections.sort(cars, Comparator.comparingInt(Car::getPosition));
-        return cars.get(this.getSize() - 1);
+    private static void checkCarNameDuplicated(List<String> carNames) {
+        long count = carNames.stream().distinct().count();
```

Comment:
```suggestion
        long count = carNames.stream()
                .distinct()
                .count();
```

체이닝 메서드를 사용할 경우 이렇게 적절하게 `.` 마다 개행을 해주면 보다 코드를 이해하기 쉽습니다!!